tests/test_image.py

```diff
@@ -0,0 +1,136 @@
+import urllib.error
+import urllib.request
+from io import BytesIO
+from math import ceil
+from unittest import mock
+
+from docx.shared import Inches
+from PIL import Image
+
+from html2docx.image import USABLE_HEIGHT, USABLE_WIDTH, image_size, load_image
+
+from .utils import PROJECT_DIR, TEST_DIR
+
+broken_image = PROJECT_DIR / "html2docx" / "image-broken.png"
+broken_image_bytes = broken_image.read_bytes()
+DPI = 72
+
+
+class TestLoadImage:
```

Comment:
You can drop this class it is not required with pytest and it contains no helper functions.